src/dbnode/persist/types.go

```diff
@@ -224,6 +225,7 @@ type NoOpColdFlushNamespace struct{}
 func (n *NoOpColdFlushNamespace) OnFlushNewSeries(
 	shard uint32,
 	blockStart time.Time,
+	firstWriteAt time.Time,
```

Comment:
I think it might make sense at this point in time to turn these arguments into a struct so that it doesn't require an update at each callsite when we add further parameters.

i.e.
```go
// OnFlushNewSeriesEvent is the fields related to a flush of a new series.
type OnFlushNewSeriesEvent struct {
	Shard uint32
	BlockStart time.Time
	ID ident.ID
	Tags ident.Tags
	FirstWriteAt time.Time
}
```